project/project/PlayCrossCompilation.scala

```diff
@@ -3,10 +3,7 @@ import uk.gov.hmrc.playcrosscompilation.PlayVersion._
 
 object PlayCrossCompilation extends AbstractPlayCrossCompilation(defaultPlayVersion = Play27) {
   val playRevision: String = PlayCrossCompilation.playVersion match {
-    case Play27 => "2.7.5"
-    case Play26 => "2.6.23"
-    case Play25 => throw new IllegalArgumentException("Play 2.5 is no longer supported by this library")
+    case Play26 => "2.6.20"
+    case Play27 => "2.7.4"
```

Comment:
Has the `PlayCrossCompilation.playVersion` enum had Play 2.5 removed?

Reply:
Added back just in case someone tries to compile for Play 2.5.